apps/web/Containerfile

```diff
@@ -45,6 +44,8 @@ RUN turbo run build --filter=cuiloa-app
 
 # Run container
 FROM alpine AS runner
+# Ensure that SSL certs are present for DB connections
+RUN apk add --no-cache ca-certificates
```

Comment:
L48 contains the resolution, permitting SSL-required db connections in the runtime image.